### DISCUSSION TITLE
SQL QUERY for generating new tokens

### DISCUSSION THREAD

**Discussion by bee1850** [THI-CSI/discord-bot#10] [Ideas]
INSERT INTO \`tokens\`(\`token\`, \`targetRole\`, \`comment\`, \`createdAt\`) VALUES (SUBSTRING(MD5(RAND()), 1, 6), ROLEID, COMMENT,CURRENT_TIMESTAMP);

---
Where ROLEID is a STRING and COMMENT is a String aswell

**Comment by bee1850** [comment]
@Ampferl @THI-creator

**Comment by Ampferl** [comment]
Okay that's fine 👍🏽

**Comment by bee1850** [comment]
INSERT INTO tokens (token, targetRole, comment, createdAt)
VALUES (SUBSTR(REPLACE(UUID(), '-', ''), 1, 6), ?, ?, NOW())
